Makefile

```diff
@@ -37,13 +37,7 @@ K3S_SELINUX_VERSION ?= 0.4-1
 .PHONY: build
 build:
 	-mkdir -p ./bin
-	cp Dockerfile ./bin/Dockerfile
-	CGO_ENABLED=0 go build -o ./bin ./cmd/proxy-server/
 	CGO_ENABLED=0 go build -o ./bin ./cmd/karavictl/
-	CGO_ENABLED=0 go build -o ./bin ./cmd/sidecar-proxy/
-	CGO_ENABLED=0 go build -o ./bin ./cmd/tenant-service/
-	CGO_ENABLED=0 go build -o ./bin ./cmd/role-service/
-	CGO_ENABLED=0 go build -o ./bin ./cmd/storage-service/
```

Comment:
Why are build targets for the services being removed?

Reply:
The service gets built inside the Golang image and then copied into the final image.

https://github.com/dell/karavi-authorization/blob/50872f0647daae010002adb1a65d4e04ee7a45a4/Dockerfile#L26
https://github.com/dell/karavi-authorization/blob/50872f0647daae010002adb1a65d4e04ee7a45a4/Dockerfile#L38